mmv1/products/securitycenter/api.yaml

```diff
@@ -18,6 +18,9 @@ versions:
   - !ruby/object:Api::Product::Version
     name: ga
     base_url: https://securitycenter.googleapis.com/v1/
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://securitycenter.googleapis.com/v1beta1/
```

Comment:
I'm assuming this was added to try to get conditions to work? Or is there another reason?

This seems to cause some other tests to fail - the v1beta1 API is missing a number of endpoints, including the entire notificationconfig resource. This also shouldn't be required for adding IAM support; conditions should already be GA (assuming they even work with this API - I'll look into it.)

Reply:
Yeah, this was added to see if that supports conditions. I removed it.

Reply:
For reference, it looks from the docs like conditions are supposed to be supported: https://cloud.google.com/security-command-center/docs/reference/rest/Shared.Types/Policy so if they're not working it's either an API bug or a docs issue.

b/246645013